hybrid/include/utils.h

```diff
@@ -0,0 +1,25 @@
+/* Inline Assembly utility functions to read/write to/from
+ * the Default Data Capability Register (DDC)
+ */
+
+void write_ddc(void *__capability cap);
+void *__capability read_ddc();
+
+/* Write a capability to the DDC
+ */
+inline void write_ddc(void *__capability cap)
+{
+	asm("MSR DDC, %[cap]\n\t"
```

Comment:
Should this function/file be guarded as only compileable on Morello?

Reply:
Yes, better here than in every file.

Reply:
Fix in fea1493.